Packs/Vectra_AI/Scripts/VectraDetectDisplayDetections/VectraDetectDisplayDetections.py

```diff
@@ -0,0 +1,115 @@
+import json
+import copy
+import traceback
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+""" CONSTANTS """
+
+UTM_PIVOT = f"?pivot=Vectra_AI-XSOAR-{get_pack_version(pack_name='Vectra AI') or '2.0.0'}"
+
+
+def trim_api_version(url: str) -> str:
+    """
+    Trim the '/api/v2.5' portion from a URL.
+
+    Args:
+        url (str): The URL to trim.
+
+    Returns:
+        str: The trimmed URL.
+    """
+    if 'pivot' in url:
+        return url
+    api_versions = ["/api/v2.5", "/api/v2"]
+    for api_version in api_versions:
+        if api_version in url:
+            trimmed_url = url.replace(api_version, "") + UTM_PIVOT
+            return trimmed_url
+    return url
+
+
+def convert_to_string(value):
+    """
+    A function that recursively converts the input value to a string.
+    If the value is a dictionary, it recurses into the dictionary and converts all values to string.
+    If the value is a list, it recurses into the list and converts all items to string.
+    For any other type, it converts the value to a string.
+
+    Args:
+        value: The value to be converted to a string.
+
+    Returns:
+        The converted value as a string.
+    """
+    if isinstance(value, dict):
+        # Recurse into dictionaries
+        return {k: convert_to_string(v) for k, v in value.items()}
+    elif isinstance(value, list):
+        # Recurse into lists
+        return [convert_to_string(item) for item in value]
+    else:
+        # Convert everything else to string
+        return str(value)
+
+
+def get_detections_list_hr(detections) -> CommandResults:
+    """
+    Convert a list of detections into a human-readable format for display.
+
+    Args:
+        detections (list): A list of detections, each represented as a JSON string.
+
+    Returns:
+        CommandResults: An object containing the human-readable output of the detections.
+    """
+    hr_dict = []
+    detection_list = copy.deepcopy(detections)
+    if not bool(detection_list) or not json.loads(detection_list[0]):
```

Comment:
```suggestion
    if not detection_list or not json.loads(detection_list[0]):
```

Reply:
`not []` is True, no need to covert it to boolean

Reply:
Updated.